src/components/SpeakerCard/SpeakerCard.scss

```diff
@@ -0,0 +1,78 @@
+@use "~stylesheets/typography";
+
+.SpeakerCard {
+  padding: 4% 5%;
+  margin: auto;
+
+  &__firstLayer {
+    width: 18em;
+    background: #ffd9e0;
+    border-radius: 72px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```

Comment:
Because each child component is defined with `width: 100%`, this actually doesn't do anything, so we can remove it.